snap/snapcraft.yaml

```diff
@@ -102,11 +102,22 @@ parts:
         - pkg-config
 
   chisel:
-    plugin: go
+    plugin: nil
     source: https://github.com/canonical/chisel.git
     #source-commit: f0bff5a30dfdcb400b
     build-snaps:
       - go/1.18/stable
     override-build: |
       go mod download
-      go install -p "1" ./...
\ No newline at end of file
+      CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -o chisel ./cmd/chisel
```

Comment:
btw, I don't think GOARCH should be hardcoded, but rather left to default into the snapcraft's `architecture.build-for`